src/lib/y2users/linux/writer.rb

```diff
@@ -216,6 +217,24 @@ def change_password(user, issues)
         set_password_attributes(user, issues)
       end
 
+      # Writes authorized keys for given user
+      #
+      # @see Yast::Users::SSHAuthorizedKeyring#write_keys
+      #
+      # @param user [Y2User::User]
+      # @param issues [Y2Issues::List] a collection for adding issues if something goes wrong
+      def write_auth_keys(user, issues)
+        return unless user.home
```

Comment:
NP: I would log this case: "Authorized keys cannot be written because the user has no home".

Reply:
Think that at this point we don't know if there are keys to being written. That's the job of SSHAuthorizedKeyring#write_keys, which first _clean_ the old ones and then writes the new ones, if any. But it crash when _home_ is not defined (because a nil.to_s)

Perhaps it should do nothing if home is nil

Reply:
I expect that `user.authorized_keys == initial_user.authorized_keys` for the system users, so I think this method is not called for them.